control-base/src/main/java/fi/nls/oskari/control/data/InspireThemesHandler.java

```diff
@@ -24,16 +24,16 @@
 @OskariActionRoute("InspireThemes")
 public class InspireThemesHandler extends RestActionHandler {
 
-    private InspireThemeService inspireThemeService;
+    private OskariMapLayerGroupService oskariMapLayerGroupService;
```

Comment:
The name of this ActionRoute should be updated. However this might not be the PR for that.